handler/src/main/java/io/netty/handler/ssl/OpenSslSessionContext.java

```diff
@@ -97,7 +97,10 @@ public void setTicketKeys(byte[] keys) {
     }
 
     /**
-     * Sets the SSL session ticket keys of this context.
+     * Sets the SSL session ticket keys of this context. Depending on the underlying native library you may omit the
+     * argument or pass an empty array and so let the native library handle the key generation and rotating for you.
```

Comment:
Not necessary now, but when client will support session tickets we can clarify that non-empty keys param is changing behavior only for the server side.

Reply:
yeah lets tackle this once we have everything in place